lockup/README.md

```diff
@@ -1,3 +1,371 @@
 # Lockup / Vesting contract
 
-*Work in progress.*
+## Overview
+
+This contract acts as an escrow that locks and holds owner's funds for the lockup period. The lockup period either starts
+at the given timestamp or from the moment transfers are enabled by voting.
+If transfers are not enabled yet, the contract keeps the account ID of the transfer poll contract.
+When the transfer poll is resolved, it returns the timestamp when it was resolved and it's used as the beginning of the
+lockup period.
+
+Once all funds are unlocked (including vesting) and transfers are enabled, the owner can add a full access key to the
```

Comment:
Can this happen anytime after the cliff, but the funds are prorated?

Reply:
updated to say that everything has to be vested.